src/script/auth/component/PhoneLoginForm.tsx

```diff
@@ -77,7 +72,11 @@ const PhoneLoginForm = ({isFetching, onSubmit}: LoginFormProps) => {
     <div>
       <Select
         id="select-phone"
-        onChange={onSelectCountry}
+        onChange={country => {
+          const selectedCountryValue = country?.value as string;
```

Comment:
i still prefer but i wont block this
```suggestion
          const selectedCountryValue = country?.value.toString();
```